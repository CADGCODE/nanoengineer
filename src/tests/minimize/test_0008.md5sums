stdout 5C3DE247AB2C6F7034EBAC40B9B63DD8
stderr D41D8CD98F00B204E9800998ECF8427E
test_0008.xyz 18B9F7B70575CEA39405604A1F24D9B5
