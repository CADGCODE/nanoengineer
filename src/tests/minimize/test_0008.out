======= test_0008.test =======
# test ground in minimize
======= exitvalue =======
0
======= stderr =======
Warning: minimization terminated after 100 iterations
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0008.mmp  > test_0008.xyz

Total Ke = 8.694000e-20
final RMS gradient=2734.070312 after 100 iterations
======= test_0008.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0008.mmp
# Output File: test_0008.xyz
# Trace File: test_0008.trc
# Number of Atoms: 14
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 1 columns:
# Ground.2: torque (nn-nm)
#

# rms force, high force
1 9433.16 21438.60
2 8483.38 19034.43
3 7660.84 16684.46
4 6819.85 14596.31
5 5809.27 12407.56
6 4770.82 11355.53
7 7158.06 18042.33
8 141118.47 341732.06
9 214318.91 571604.65
10 68129.25 156813.10
11 34952.44 71561.79
12 19341.24 35532.70
13 51309.73 121047.22
14 26428.26 52376.50
15 23754.87 41345.61
16 64255.26 158787.17
17 36297.99 76542.10
18 22236.70 46533.59
19 20330.28 47003.57
20 16674.69 38699.48
21 40253.52 101882.56
22 19855.42 55773.57
23 15863.16 43095.78
24 9254.60 15902.59
25 9954.50 22504.56
26 65519.83 172166.73
27 18557.26 54665.46
28 16468.88 48407.49
29 10448.54 27121.09
30 8735.48 18578.22
31 8821.19 18853.43
32 10984.30 28480.07
33 7408.18 17726.30
34 4973.15 10933.94
35 3699.67 8837.90
36 4069.07 8822.68
37 15187.60 41292.61
38 6333.14 17310.57
39 4983.82 11704.49
40 3062.97 8781.78
41 2833.03 8766.83
42 2958.21 8751.01
43 6185.90 15968.20
44 13359.96 35276.17
45 9128.44 23837.56
46 12063.74 29998.49
47 7741.46 20300.82
48 5017.66 10304.89
49 4102.30 8867.20
50 4364.48 9314.69
51 4110.73 9037.37
52 3353.24 8578.73
53 3197.81 8564.96
54 4272.40 8773.00
55 8358.91 21795.06
56 5782.19 13508.46
57 3179.29 8533.71
58 2896.06 8528.48
59 3090.94 8509.58
60 5599.65 10484.61
61 10984.40 26051.52
62 5461.44 11014.58
63 3502.31 8456.04
64 2830.89 8459.15
65 2829.26 8444.04
66 3230.69 8444.04
67 6545.09 16034.78
68 19747.48 43580.44
69 5700.67 10514.16
70 4195.42 8412.94
71 2994.08 8385.14
72 2734.07 8372.60
73 2784.05 8366.78
74 3470.79 8332.87
75 30352.15 83176.05
76 9801.53 27301.34
77 6052.01 16305.30
78 4426.88 9627.00
79 4355.98 9818.12
80 8433.60 22657.96
81 6732.06 18696.87
82 3978.96 8244.84
83 3347.19 8253.81
84 3666.09 8227.67
85 6718.92 17633.11
86 5671.91 14222.00
87 3049.94 8208.13
88 2880.17 8207.67
89 3233.42 8188.63
90 7477.39 18177.77
91 9609.19 23724.96
92 3532.93 8173.63
93 2809.38 8154.46
94 2797.59 8149.52
95 3342.76 8137.74
96 10674.21 27810.48
97 9924.63 23134.93
98 6392.63 14419.05
99 4060.75 8083.65
100 2734.07 8372.60
# Warning: minimization terminated after 100 iterations
# Done: Minimization final rms: 2734.070312, highForce: 8372.599119
======= test_0008.xyz =======
14
RMS=2734.070312
C -2.732213 -0.370132 0.937678
X -2.677510 -0.920965 1.509461
X -2.731993 -0.742856 0.239516
C 0.540947 0.707096 0.956690
X 0.509418 1.198511 0.338226
X 0.526040 1.117901 1.628680
C 3.476000 0.155000 0.889000
X 3.758264 0.243657 1.633777
X 3.815674 0.683232 0.391085
X 3.586865 -0.594856 0.628819
C -7.198000 0.712000 1.277000
X -7.257445 1.510376 1.313588
X -7.485492 0.373566 1.944125
X -7.572804 0.427336 0.628338
